### PR TITLE
Traversing a graph

### PR DESCRIPTION
**Pull request by AndraB90**

### CODE REVIEW

File: src/graphs/graphs_traverse/Graph.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Queue;
+public class Graph {
+    ArrayList<Node> nodes;
+    int[][] matrix;
+
+    Graph(int size){
+
```
Comment (GabrielCorlan) [inline src/graphs/graphs_traverse/Graph.java:12]: unnecessary extra line

File: src/graphs/graphs_traverse/Graph.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        nodes = new ArrayList<>();
+        matrix = new int[size][size];
+    }
+    public void addNode(Node node) {
+
```
Comment (GabrielCorlan) [inline src/graphs/graphs_traverse/Graph.java:17]: same

File: src/graphs/graphs_traverse/Graph.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        nodes.add(node);
+    }
+    public void addEdge(int src, int dst) {
+
```
Comment (GabrielCorlan) [inline src/graphs/graphs_traverse/Graph.java:21]: same

File: src/graphs/graphs_traverse/Graph.java (hunk 2 above)
Comment (AndraB90) [reply src/graphs/graphs_traverse/Graph.java:17]: resolved

File: src/graphs/graphs_traverse/Graph.java (hunk 3 above)
Comment (AndraB90) [reply src/graphs/graphs_traverse/Graph.java:21]: Resolved. Thank you for checking my code :)

**Pull request merged**